tests/service/certificate/test_service_certificate.yml

```diff
@@ -76,7 +76,7 @@
       ok_as_delegate: no
       ok_to_auth_as_delegate: no
     register: result
-    failed_when: not result.changed
+    failed_when: not result.changed and result.failed
```

Comment:
This change is unrelated.

Reply:
This change is in a TEMP commit, so service tests pass.